test/gtest/ucp/ucp_test.cc

```diff
@@ -297,17 +330,34 @@ ucs_status_t ucp_test::requests_wait(std::vector<void*> &reqs,
     return ret_status;
 }
 
+ucs_status_t
+ucp_test::requests_wait(const std::initializer_list<void*> reqs_list,
+                        int worker_index)
+{
+    std::vector<void*> reqs(reqs_list);
+    return requests_wait(reqs, worker_index);
+}
+
 void ucp_test::request_release(void *req)
 {
     request_process(req, 0, false);
 }
 
-void ucp_test::wait_for_wakeup(const std::vector<ucp_worker_h> &workers,
-                               int poll_timeout, bool drain)
+void ucp_test::request_cancel(entity &e, void *req)
+{
+    if (UCS_PTR_IS_PTR(req)) {
```

Comment:
IMO it should be assert

Reply:
then we will have to wrap each call by `if` condition

Reply:
it is aligned with `request_process()` which is used for `request_wait()`

Reply:
> then we will have to wrap each call by `if` condition

it depends on usage, usually we have to do it anyway to get status.

> it is aligned with request_process() which is used for request_wait()

ok then